tests/ops/op_math/test_sum.py

```diff
@@ -497,14 +497,14 @@ def test_eigendecompostion(self):
 
         true_eigvecs = qnp.tensor(
             [
-                [0.0, 0.0, 1.0, 0.0],
-                [0.0, 0.0, 0.0, 1.0],
                 [1.0, 0.0, 0.0, 0.0],
                 [0.0, 1.0, 0.0, 0.0],
+                [0.0, 0.0, 1.0, 0.0],
+                [0.0, 0.0, 0.0, 1.0],
             ]
         )
 
-        true_eigvals = qnp.tensor([0.0, 0.0, 2.0, 2.0])
+        true_eigvals = qnp.tensor([2.0, 2.0, 0.0, 0.0])
```

Comment:
The `eig` function does not order the eigenvalues or vectors in any particular consistent manner